src/stores/SpaceStore.tsx

```diff
@@ -530,6 +532,14 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
             });
         }
 
+        const hiddenChildren = new EnhancedMap<string, Set<string>>();
```

Comment:
Suggestion: I found the name `hiddenChildren` a bit misleading as, if I understand correctly, it contains a { [ parent ]: visibleRoom } mapping for all rooms in `visibleRoom` - which is done for all visible rooms, not just for hidden ones .. maybe a name like `visibleRoomParents` or `parentsToVisibleRooms` would be closer to what it is?

Reply:
Right but it is a map from `Parent -> Set<HiddenChild>`, so is the list of hidden children, per parent.